optimizer-plus/optimizer-plus-sd-policy/base-policy.tf

```diff
@@ -0,0 +1,121 @@
+provider "google" {
+  project = var.project_id
+}
+
+resource "google_monitoring_alert_policy" "uptime_check" {
+  display_name = "rax-optimizer-plus-monitoring-instance-uptime-check"
+  combiner     = "OR"
+  enabled      = true
+  conditions {
+    display_name = "Uptime check for GCE INSTANCE - Platform"
+    condition_threshold {
+      filter     = <<EOT
+              metric.type="compute.googleapis.com/instance/uptime" AND
+              metadata.user_labels.autoscaled="false" AND
+              metadata.user_labels.monitored="true" AND
+              resource.type="gce_instance"
+      EOT
+      duration   = "900s"
```

Comment:
Could possibly be reduced, 5 minutes is usually the maximum amount of time it takes an instance to go down and come back up. Anything longer is going to be hard down, or boot issues.